src/foam/nanos/approval/ApprovableAwareDAO.js

```diff
@@ -135,8 +135,12 @@ foam.CLASS({
       }
 
       if ( getIsTrackingRequestSent() ){
-        sendSingleRequest(x, request, request.getInitiatingUser());
-        approverIds.remove(request.getInitiatingUser());
+        RoleApprovalRequest trackingRequest = (RoleApprovalRequest) request.fclone();
```

Comment:
Below ApprovalRequest tests tracking, but here we assume RoleApprovalRequest.

Reply:
RE: my comment about moving the extra properties isFulfilled and Operation to approvalRequest and changing initiatingUser to createdBy.

Reply:
We can merge this in now so it fixes and ill do the migration in another PR @jlhughes ?